### PR TITLE
Make lxml and bs4 optional

### PR DESCRIPTION
**Pull request by dylanmccall**
These are unavailable on some platforms which are support by Kolibri.

https://phabricator.endlessm.com/T33395

### CODE REVIEW

File: kolibri_zim_plugin/assets/src/views/ZimSearchView.vue [hunk 1]
```diff
@@ -68,7 +68,9 @@
                 }
               ) }}
             </small>
-            <p>{{ article.snippet }}&nbsp;&hellip;</p>
+            <p v-if="article.snippet">
+              {{ article.snippet }}&nbsp;&hellip;
+            </p>
```
Comment (manuq) [inline kolibri_zim_plugin/assets/src/views/ZimSearchView.vue:73]: :+1:

File: Pipfile [hunk 2]
```diff
@@ -6,11 +6,11 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-beautifulsoup4 = "==4.9.3"
+beautifulsoup4 = "==4.11.1"
 bumpversion = "*"
 flake8 = "==3.8.3"
 ipdb = "==0.13.2"
-lxml = "==4.6.3"
+lxml = "==4.9.0"
```
Comment (manuq) [inline Pipfile:13]: :+1: good to have these deps updated

**Review by danigm — APPROVED** [review]
Looks good to me.

**Review by manuq — APPROVED** [review]
Well I was reviewing but it's already merged :)

**Pull request merged**